### PR TITLE
add natural england as an approved mail domain 

### PR DESCRIPTION
**Pull request by adam-moss**

### CODE REVIEW

**Comment by bruntonspall** [comment]
I was a bit scared of this, and the resultant regex.  Could you rebuild against the new refactored code, including tests and then I'll merge it.

File: slack_bot.js [hunk 1]
```diff
@@ -342,7 +342,7 @@ controller.hears(["^set role for <@(.*)> to (.*)"], 'direct_mention,direct_messa
 });
 
 function hasApprovedEmailDomain(email) {
-  if (email.match(".*\.gov\.uk$")) {
+  if (email.match(".*(\.gov|naturalengland\.org){1}\.uk$")) {
```
Comment (alexbishop1) [inline slack_bot.js]: This will accept bob@idontworkfornaturalengland.org.uk — should be `.*\.(gov|naturalengland\.org)\.uk$` or something

**Comment by adam-moss** [comment]
I think I like this regex even less 🤒 

https://regex101.com/r/TwfJeB/1

**Comment by bruntonspall** [comment]
This was fixed in the rewrite

**Pull request closed without merging**